twisted-infra/braid
# Move mailman submodule as a normal folder

Scope
=====

We want to move all twisted-infra services as normal folders to braid and don't use submodules

Changes
=======

Removed mailmain submodule, imported the code from https://github.com/twisted-infra/mailman-config and updated the README.

I had to run package.update() before install so that the right packages are installed.

No need to migrate issues or PR to braid.
I have update the repo description for twisted-infra/mailman-config 
Future work
=========

We might want to check that production mailman data is properly backed up and maybe update the fabfile.py to set up a cron to backup and store the data in on off site location.


How to test
=========

Check that you can run the command on the vagrant VM
```
fab -i ~/.vagrant.d/insecure_private_key config.vagrant mailman.install
fab -i ~/.vagrant.d/insecure_private_key config.vagrant mailman.dump:dump.tar
fab -i ~/.vagrant.d/insecure_private_key config.vagrant mailman.restore:dump.tar
```

Thanks!
